requirements.txt

```diff
@@ -0,0 +1,6 @@
+ibapi
+jpy
+deephaven
```

Comment:
a naive pip install -r will pick up the DHE version on PYPI?

Reply:
Probably not yet.  How should we handle this?  Comment out `jpy` and `deephaven`?